fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowProcesslistStmt.java

```diff
@@ -12,9 +12,10 @@
 public class ShowProcesslistStmt extends ShowStmt {
     private static final ShowResultSetMetaData META_DATA =
             ShowResultSetMetaData.builder()
+                    .addColumn(new Column("FeHost", ScalarType.createVarchar(16)))
```

Comment:
This is a command from MySQL.
I think the output should be the same as MySQL's. Then other BIs or management tools can recognize the results.

Reply:
Actually, the META_DATA  has already add two new column ConnectionStartTime and IsPending which different from mysql protocol.

Reply:
And, If no new column is added, it is impossible to determine which frondend  the session belongs to

Reply:
@imay take a look? how to deal with this？

Reply:
how about put FeHost at the end of all columns?

Reply:
Named FeHost will be more easier to understand?